README.md

```diff
@@ -2,11 +2,59 @@
 title: Firebolt OpenRPC Tools
 ---
 # Firebolt OpenRPC Tools
-Tools to enable consistent Firebolt SDK generation.
+Tools to enable consistent Firebolt SDK and document generation.
 
-## Usage
-To install the package, run:
+## To Use
 
-```
-npm install @firebolt-js/openrpc
-```
\ No newline at end of file
+  - First, `npm install`.
+  - Next, invoke the cli directly with `src/cli.js`. See below for arguments.
+
+Alternatively, you may install globally using `npm install -g @firebolt-js/openrpc`, which will install the cli and add it to your $PATH so it may be invoked with the `firebolt-openrpc` command. However, trying this at the time of writing, it did not work.
+
+## CLI Arguments
+
+`--task`: Tell the tool what you want it to do. Options are:
+
+  - `sdk`: Generate the SDK
+  - `docs`: Generate the docs
+  - `openrpc`: Generate the openrpc
+  - `validate` Validate the openrpc
+
+`--source`: The relative or absolute path to the folder or file that the task will use.
+
+`--template`: The relative or absolute path to the folder or file containing template resources for the task.
+
+`--output`: The relative or absolute path to the folder or file that will hold the task's generated output.
+
+`--as-path`: This has some effect on the document generation task. Not sure what.
+
+`--static-modules`: Used by the SDK generator. Makes sure that statically defined modules get exported. An example is `Platform`.
```

Comment:
Might be worth mentioning that a statically defined module is one w/out an open-rpc .json document, that is included statically in the template directory. Listing it in this property ensures it will be exported by the SDK, and it will have mock responses wired up as well.